src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php

```diff
@@ -189,7 +189,7 @@ public function postAction(Request $request)
      * Change a filter by the given id.
      *
      * @param \Symfony\Component\HttpFoundation\Request $request
-     * @param int $id the attribute id
+     * @param int                                       $id      the attribute id
```

Comment:
Actually we don't do that, that's also the reason we've put that into our `.styleci.yml`: https://github.com/sulu-io/sulu/blob/develop/.styleci.yml#L10

If they are already aligned that was not intended. Has the configuration file maybe also changed?